src/Plugin/EndpointInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\relaxed\Plugin\EndpointInterface.
+ */
+
+namespace Drupal\relaxed\Plugin;
+
+use Drupal\Component\Plugin\ConfigurablePluginInterface;
+use Psr\Http\Message\UriInterface;
+
+/**
+ * Defines an interface for Endpoint plugins.
+ */
+interface EndpointInterface extends ConfigurablePluginInterface, UriInterface  {
+
+  // Add get/set methods for your plugin type here.
```

Comment:
Should this be a proper `// @todo`? That way we can easily grep or search for all todos in the code. This one will silently disappear otherwise ;)